controllers/virtualmachinepublishrequest/virtualmachinepublishrequest_controller.go

```diff
@@ -840,28 +875,9 @@ func (r *Reconciler) ReconcileNormal(ctx *context.VirtualMachinePublishRequestCo
 		vmopv1alpha1.VirtualMachinePublishRequestConditionComplete); isComplete {
 		requeueAfter, deleted, err := r.removeVMPubResourceFromCluster(ctx)
 		isDeleted = deleted
-		return ctrl.Result{RequeueAfter: requeueAfter}, err
+		return ctrl.Result{RequeueAfter: requeueAfter}, false, err
```

Comment:
Should we return true here to skip the patch since there is no update of the resource in `removeVMPubResourceFromCluster`?

Reply:
Will keep it as `false` based on below comments.